demisto_sdk/commands/prepare_content/markdown_images_handler.py

```diff
@@ -89,6 +107,12 @@ def replace_markdown_rel_paths_and_upload_to_artifacts(
     Returns:
         - A dict in the form of {pack_name: [images_data]} or empty dict if no images relative paths were found in the README
     """
+    lock_file_path = f"{os.getenv('ARTIFACTS_FOLDER')}/{MARKDOWN_RELATIVE_PATH_IMAGES_ARTIFACT_FILE_NAME.replace('json', 'lock')}"
+    wait_for_lock_reopen(
+        lock_file_path,
+        init_json_file,
+        {"markdown_images_file_name": MARKDOWN_RELATIVE_PATH_IMAGES_ARTIFACT_FILE_NAME},
+    )
```

Comment:
for readability, I'd wrap this with something like:
```
safe_init_json_file(MARKDOWN_RELATIVE_PATH_IMAGES_ARTIFACT_FILE_NAME)
```
same for `update_markdown_images_file_links`